onset_maproom/cess.py

```diff
@@ -0,0 +1,133 @@
+#temporary file for cessation date functions; 
+#these will be moved to the main calc.py file when finished
+
+import calc
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+#params for cess as found in maproom
+search_start_day = 1
+search_start_month = 9
+search_days = 90
+dry_thresh = 5
+min_dry_days = 3
+time_coord = "T"
+et = 5
+taw = 60
+sminit = 0 
+
+rr_mrg = calc.read_zarr_data("/data/aaron/ethiopia-rain-rechunked")
+
+#subset of data for testing
+rr_mrgSub = rr_mrg.sel(T=slice('1981-01-01','1983-12-31'))
+daily_rain = rr_mrgSub.isel(X=33,Y=3)
+
+
+#onset date calc used in seasonal cess date func
+#first date after START_DAY(1), START_MONTH(9) in SEARCH_DAYS(90) 
+#days when the soil water balance falls below DRY_THRESH(5)mm 
+#for a period of DRY_SPELL_LENGTH(3) days
+def cess_date(daily_rain, dry_thresh, min_dry_days, et, taw, sminit, time_coord="T"):
+    print(daily_rain)
+    daily_rain = daily_rain.to_array()
```

Comment:
According to the docs, `to_array` broadcasts all of the variables against each other. That's not what you want. As I said in the other comment, if you want to extract a single variable from a Dataset, just use `ds["varname"]`.